parsing/pprintast.ml

```diff
@@ -2070,11 +2084,7 @@ and type_extension ctxt f x =
     pp f "@\n|@;%a" (extension_constructor ctxt) x
   in
   pp f "@[<2>type %a%a += %a@ %a@]%a"
-    (fun f -> function
-       | [] -> ()
-       | l ->
-           pp f "%a@;" (list (type_param ctxt) ~first:"(" ~last:")" ~sep:",") l)
```

Comment:
The line-break behavior is different here - the break comes before the comma, whereas in the other case it comes after. But with this PR, it's now always after. Maybe this is fine, but could you add a test case that shows what it looks like with line breaks?

Reply:
Looks OK to me.